File: U3P2C#/U3P_17.cs
﻿using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class U3P_17
    {
        static void Main(string[] args)
        {
            char opcion;
            do
            {
                Console.WriteLine("Bienvenido a una sola ejecucion");
                Console.WriteLine("¡¡¡Escribe n para salir!!!");
                opcion = Console.ReadKey().KeyChar;
            } while (opcion != 'n');
        
        }
    }
}

File: U3P2J/U3P_17.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u3p_17;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class U3P_17 {

    public static void main(String[] args) {
        Scanner sc = new  Scanner(System.in);
        char opcion;
        do{
        System.out.println("Bienvenido a una sola ejecucion");
        System.out.println("¡¡¡Escribe n para salir!!!");
        opcion=sc.next().charAt(0);//1 CARACTER
        }while(opcion !='n');
    }
}
